notepad2e.txt

```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = Notepad2e
+Version = 4.2.25
+License = 3-clause BSD
+Description = Notepad 2e is a fork of Notepad2 with word highlighting, simultaneous editing, math evaluation, un/grep, UAC elevation, complete regexps (PCRE), Lua lexers, DPI awareness and more 
+Size = 980 KiB
```

Comment:
Size is not needed when SizeBytes is used.

Reply:
This should be mentioned in [the wiki](https://reactos.org/wiki/RAPPS).